docs/api/ethereum.md

```diff
@@ -1,5 +1,645 @@
 # API / Ethereum
 
+## Bitski back-end provider
+
+A [Bitski](https://www.bitski.com/) back-end provider is applied back-end Node.js usage. You will be able to connect to ethereum blockchain and create queries and mutation via your Bitski developer wallets. To setup Bitski back-end provider it is required to first create a [development account](https://developer.bitski.com/) on Bitski.
+
+### BitskiProvider(options)
+
+A `class` providing the communication with the Ethereum blockchain through [Bitski](https://www.bitski.com/).
```

Comment:
`A `class` providing communication with the Ethereum blockchain through [Bitski](https://www.bitski.com/).`